### ISSUE TITLE
Are browser console data used for triage and diagnosis?

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2995]
So we have now the console.log feature for already one year. October 16, 2018
https://github.com/webcompat/webcompat.com/issues/2653

The question is: "do we use the report for the purpose of triaging or diagnosing?"

@cipriansv @softvision-oana-arbuzov @softvision-sergiulogigan 
@denschub @ksy36 @miketaylr @wisniewskit

**Comment by karlcow** [comment]
In the last year, I have used it maybe once and it was not conclusive. 
I'm not sure it's because it is not useful as it is, or just because I forget about it in the UI.

So I was wondering that if I was not alone, maybe we should just scrap it. But if it's useful for others then let's make it better.

**Comment by wisniewskit** [comment]
Yes. I even used it today. It helps me immediately confirm that I'm seeing the same issue that is being reported, when a screenshot cannot help. It also helps me make a suggestion when I *can't* reproduce an issue, especially when it might be due to an ad I am not seeing breaking a page.

With that said, I cannot confirm that I use it enough to make it worth keeping, so if there is a good reason to drop it then we should discuss.

**Comment by wisniewskit** [comment]
Another good question here is: how *would* we improve it? Do you mean in the sense of making it easier to remember that it exists, or in general?

Because I think that aside from improving its presentation and basic UX, it's unclear how we could improve it aside from adding more details to it. I feel there are quite a few things we can do to improve the UX, but they may not be worth the effort:

- make it more obvious when there is anything to look at (discoverability).
- use a list/table to make it easier to parse.
- use color-coding where we can (yellow for warnings vs red for errors, etc).
- make links clickable (open a given script in a tab).
- add filtering/sorting/grouping if we're running into long enough logs.

However I'm not as sure what kinds of additional data we might want, or have access to. That effort might actually be best invested at the Gecko-level, to imrpove the messages we're logging (for instance, to give accurate column+line numbers to things like stylesheet warnings).

Maybe we could access some of the newer webcompat messages/details that are being added to the devtools?

**Comment by karlcow** [comment]
Thanks a lot @wisniewskit 
To note that @ksy36 is working on the readability part #2659

**Comment by softvision-sergiulogigan** [comment]
We don't use them, during Triage.

**Comment by miketaylr** [comment]
I use often them (when I do get around to diagnosis) just to check for obvious errors. When I go to verify reproduce / investigate, I do want to see my own errors and it's nice to be able to compare. In terms of UX, all I would need is a non-scrollable blob of plain text. There's a couple of ways that @ksy36 is exploring that in #2659.
